packaging/irods.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=iRODS Open Source Data Management Software
+After=network.target auditd.service
+
+[Service]
+EnvironmentFile=/etc/irods/service_account.config
+WorkingDirectory=/var/lib/irods
+User=irods
+Group=irods
+ExecStart=/var/lib/irods/irodsctl start
+ExecReload=/var/lib/irods/irodsctl restart
+ExecStop=/var/lib/irods/irodsctl stop
+KillMode=process
```

Comment:
This needs to be `KillMode=control-group` (or just removed, as `control-group` is the default).

Per the `systemd.kill` documentation:
> Note that it is not recommended to set `KillMode=` to `process` or even `none`, as this allows processes to escape the service manager's lifecycle and resource management, and to remain running even while their service is considered stopped and is assumed to not consume any resources.